rust/theoros/src/types/event_storage.rs

```diff
@@ -0,0 +1,31 @@
+use parking_lot::RwLock;
+use std::collections::VecDeque;
+
+use crate::types::dispatch_event::DispatchEvent;
+
+pub struct EventStorage {
```

Comment:
add rust doc to explain how we use it, meaning as a FIFO buffer of fixed size

Reply:
done sir @EvolveArt